docs/source/index.mdx

```diff
@@ -21,24 +25,46 @@ description: A strongly-typed, caching GraphQL client for iOS, written in Swift
 
-Thanks to strong typing in Apollo iOS, you don't need to deal with parsing JSON responses or passing around dictionaries of values that require manual casting. You also don't need to write model types yourself, because models are generated from the GraphQL operations your UI defines.
+Because generated response models are operation-specific, they include properties _only_ for the GraphQL fields included in their corresponding operation. This means you can rely on the Swift type checker to flag data access errors at compile time.
 
-Because generated types are operation-specific, they include properties _only_ for the GraphQL fields included in their corresponding operation. This means you can rely on the Swift type checker to flag data access errors at compile time.
+Models for your GraphQL fragments are also generated, which operation-specific models can easily be converted to. This allows you to share common models across multiple GraphQL operations.
 
-Apollo's Xcode integration enables you to work with your UI code and corresponding GraphQL definitions side by side. It even validates your query documents, showing errors inline.
+Apollo iOS also generates models for your schema's input types, enums, and custom scalars, making working with your GraphQL data in pure Swift code safe and easy.
 
-### Normalized caching
+### The Normalized Cache
 
 Apollo iOS normalizes operation results to build a client-side cache of your data, which is updated with every operation you execute. This means your UI is always internally consistent, and it can stay up to date with your backend with as few operation as possible.
 
-[Learn more about caching.](./caching/)
+Our normalized cache is built specficially with GraphQL in mind. It deduplicates GraphQL response data, merging data from separate GraphQL operations into normalized objects. This enables your app to use it as a source of truth for your data graph and it can stay up-to-date with your backend with as few operation as possible.
+
+You can also use a normalized cache as a single source of truth for your UI, enabling it to react to changes in the cache. To learn more about the normalization process, see our blog post [Demystifying Cache Normalization](https://www.apollographql.com/blog/apollo-client/caching/demystifying-cache-normalization).
+
+### Networking and GraphQL Execution
+
+Apollo iOS provides a networking API that makes it easy to use your GraphQL operations with any GraphQL spec-compliant server implementation. Our GraphQL Execution pipeline automatically caches your response data in the normalized cache and returns the response data using your generated, type-safe operation models.
+
+Apollo iOS handles all the heavy lifting of networking with GraphQL including:
+
+- Construction of GraphQL Spec-Compliant Network Requests
```

Comment:
I'm curious what our approach is for the camel casing of these terms, Some of them seem like they probably don't need capitalization.  @StephenBarlow would probably have some insights here.

Reply:
Yeah, because it’s a bulleted list, I was following Title Casing formatting. Is that wrong here?
I wasn’t capitalizing them because they were proper nouns.

Reply:
We don't really use title casing anywhere in the docs except in the expandable category titles in the left navigation